‚ö†Ô∏è  TMUX SESSION EXITED UNEXPECTEDLY AT Fri Jul 11 09:24:50 PM CEST 2025
Exit code: 0
Last 50 lines of training log:
=====================================
Mixed precision type: no

üîß Loading pretrained confidence grid from: 
2025-07-11 21:10:27: Error!
Traceback (most recent call last):
  File "/home/nilkel/Projects/zipnerf-pytorch/train.py", line 716, in <module>
    app.run(main)
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/absl/app.py", line 316, in run
    _run_main(main, args)
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/absl/app.py", line 261, in _run_main
    sys.exit(main(argv))
  File "/home/nilkel/Projects/zipnerf-pytorch/train.py", line 70, in main
    model = models.Model(config=config)
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/gin/config.py", line 1605, in gin_wrapper
    utils.augment_exception_message_and_reraise(e, err_str)
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/gin/utils.py", line 41, in augment_exception_message_and_reraise
    raise proxy.with_traceback(exception.__traceback__) from None
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/gin/config.py", line 1582, in gin_wrapper
    return fn(*new_args, **new_kwargs)
  File "/home/nilkel/Projects/zipnerf-pytorch/internal/models.py", line 69, in __init__
    self.confidence_field = ConfidenceField(
  File "/home/nilkel/Projects/zipnerf-pytorch/internal/field.py", line 34, in __init__
    self._load_pretrained_grid(pretrained_grid_path, device)
  File "/home/nilkel/Projects/zipnerf-pytorch/internal/field.py", line 55, in _load_pretrained_grid
    pretrained_logits = torch.load(pretrained_grid_path, map_location='cpu')
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/torch/serialization.py", line 1479, in load
    with _open_file_like(f, "rb") as opened_file:
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/torch/serialization.py", line 759, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/torch/serialization.py", line 740, in __init__
    super().__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: ''
  In call to configurable 'Model' (<class 'internal.models.Model'>) in scope 'train'
Traceback (most recent call last):
  File "/home/nilkel/miniconda3/envs/zipnerf2/bin/accelerate", line 8, in <module>
    sys.exit(main())
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/accelerate/commands/accelerate_cli.py", line 50, in main
    args.func(args)
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/accelerate/commands/launch.py", line 1199, in launch_command
    simple_launcher(args)
  File "/home/nilkel/miniconda3/envs/zipnerf2/lib/python3.9/site-packages/accelerate/commands/launch.py", line 785, in simple_launcher
    raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)
subprocess.CalledProcessError: Command '['/home/nilkel/miniconda3/envs/zipnerf2/bin/python3.9', 'train.py', '--gin_configs=configs/potential_triplane.gin', "--gin_bindings=Config.data_dir = '/home/nilkel/Projects/data/nerf_synthetic/lego'", "--gin_bindings=Config.debug_confidence_grid_path = ''", '--gin_bindings=Config.freeze_debug_confidence = False', '--gin_bindings=Config.binary_occupancy = False', '--gin_bindings=Config.confidence_grid_resolution = (256, 256, 256)', "--gin_bindings=Config.comment = 'binocc_unfixedoptconf_nogate_noreg_5xconf_256res'"]' returned non-zero exit status 1.

‚úÖ Training completed successfully!
‚è∞ Finished at: Fri Jul 11 09:10:28 PM CEST 2025
üìÑ Full log: ./tmux_logs/training_pt_lego_binocc_unfixedoptconf_nogate_noreg_5xconf_256res_0711_211013.log
üìÅ Check results in: exp/

‚ùå Training failed at line 76. Check logs: ./tmux_logs/training_pt_lego_binocc_unfixedoptconf_nogate_noreg_5xconf_256res_0711_211013.log
=====================================
Process list at exit:
root        1504  0.0  0.0 112248  3220 ?        Ssl  Jun03   0:00 /usr/bin/python3 /usr/share/unattended-upgrades/unattended-upgrade-shutdown --wait-for-signal
root       11054  0.0  0.0  61392  3960 ?        S    Jun03   0:00 python3 /usr/lib/software-properties/software-properties-dbus
nilkel   2076307  0.0  0.0  17312  5840 ?        Ss   21:10   0:00 tmux new-session -d -s train_pt_lego_0711_211013 -c /home/nilkel/Projects/zipnerf-pytorch
nilkel   2076319  0.0  0.0  10056  1680 pts/13   S    21:10   0:00 /bin/bash ./train_pt_tmux.sh lego binocc_unfixedoptconf_nogate_noreg_5xconf_256res
